### PR TITLE
Update minimum version to Xcode 11.0

### PR DESCRIPTION
**Pull request by jmagman**
## Description

Updated minimum version of Xcode to match LUCI builder.  Xcode 11 was released September 2019.

## Related Issues

Tool change part of https://github.com/flutter/flutter/issues/49787.

## Tests

Updated xcodeproj_test and xcode_test.

## Checklist
- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change
- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*

### CODE REVIEW

**Comment by jmagman** [comment]
Holding - do not submit until this change is given approval in https://github.com/flutter/flutter/issues/49787.

**Review by dnfield — APPROVED** [review]
LGTM.

Maybe we could avoid some of this fallout in the future by making the devicelab tests enable bitcode?

**Comment by jmagman** [comment]
> LGTM.
> 
> Maybe we could avoid some of this fallout in the future by making the devicelab tests enable bitcode?

It manifests with bitcode disabled, too. Archiving doesn't reproduce it, it needs to be validated.

**Comment by dnfield** [comment]
We could add a test that just runs the validate Xcode check against the plist right?

**Comment by dnfield** [comment]
And give it a nice clear error message too

**Comment by pawlowskim** [comment]
> > LGTM.
> > Maybe we could avoid some of this fallout in the future by making the devicelab tests enable bitcode?
> 
> It manifests with bitcode disabled, too. Archiving doesn't reproduce it, it needs to be validated.

I can confirm that during export / validation this happens.
```
** ARCHIVE SUCCEEDED **

+ xcodebuild -exportArchive -archivePath /Volumes/xxx/yyy/workspace/ccc/ios/build/vvv.xcarchive -exportOptionsPlist exportOptions.plist -exportPath /Volumes/xxx/yyy/workspace/ccc/ios/../build/vvv.ipa
2020-01-30 15:29:24.053 xcodebuild[30972:3771336] [MT] IDEDistribution: -[IDEDistributionLogging _createLoggingBundleAtPath:]: Created bundle at path '/var/folders/wt/jjmd1pm16lv1d9ntx203shw00000gp/T/Release-Integration_2020-01-30_15-29-24.047.xcdistributionlogs'.
error: exportArchive: Failed to verify bitcode in Flutter.framework/Flutter:
error: Linker option verification failed for bundle /var/folders/wt/jjmd1pm16lv1d9ntx203shw00000gp/T/FluttermQF29X/Flutter.armv7.xar (unrecognized arguments: -platform_version iOS 8.0.0 13.0.0)
```

**Review by zanderso — APPROVED** [review]

**Comment by jmagman** [comment]
gauge tests off, merging...

**Comment by jmagman** [comment]
I forgot this was blocked on devicelab mac-10 upgrading to Xcode 11 even though I made the note for myself above.  Reverted.

**Pull request merged**
